getDisplayMedia.js

```diff
@@ -8,10 +8,6 @@ import MediaStreamError from './MediaStreamError';
 const { WebRTCModule } = NativeModules;
 
 export default function getDisplayMedia(constraints) {
```

Comment:
Can you remove the (now unused) constraints argument too?

Reply:
Done.